.github/main.go

```diff
@@ -86,107 +68,18 @@ func New() *CI {
 			}),
 			WorkflowDefaults: workflow,
 		}),
-		CloudRunner: dag.Gha(dagger.GhaOpts{
-			JobDefaults: dag.Gha().Job("", "", dagger.GhaJobOpts{
-				Runner:         []string{"ubuntu-24.04"},
-				TimeoutMinutes: timeoutMinutes,
-				CloudEngine:    true,
-			}),
-			WorkflowDefaults: dag.Gha().Workflow("", dagger.GhaWorkflowOpts{
-				PullRequestConcurrency:      "preempt",
-				Permissions:                 []dagger.GhaPermission{dagger.GhaPermissionReadContents, dagger.GhaPermissionWriteIdToken},
-				OnPushBranches:              []string{"main"},
-				OnPullRequestOpened:         true,
-				OnPullRequestReopened:       true,
-				OnPullRequestSynchronize:    true,
-				OnPullRequestReadyForReview: true,
-			}),
-		}),
 	}
-
 	return ci.
-		withWorkflow(
-			ci.AltRunnerWithCache,
-			false,
-			"Check generated files",
-			"check-generated",
-		).
```

Comment:
Because this is now gone, we can remove the `start-dev-dagger.sh` script in `modules/gha` and everything that needed it.

Should be a nice simplification, the more we can strip out of that module, the closer we can see we're getting to native CI.

Reply:
Done :) That felt good